pom.xml

```diff
@@ -175,8 +178,7 @@
                     <artifactId>maven-compiler-plugin</artifactId>
                     <version>${maven.compiler.plugin.version}</version>
                     <configuration>
-                        <source>${java.version}</source>
-                        <target>${java.version}</target>
+                        <release>${maven.compiler.release}</release>
```

Comment:
@jamezp I think you forgot to remove this "release" element since the maven compiler plugin autoconfigure itself with properties `maven.compiler.release`

-- Nicolas

Reply:
Ok. So to be consistent should'nt we be that explicit here too : https://github.com/jakartaee/rest/pull/1203/files#diff-0b9d1df7da9cc5c30f2c2599dc9a182117210c78c941d576dfaca5106aa50be1R126 ?

-- Nicolas

Reply:
I only didn't there because it will be inherited from here. Again though, I have no strong opinion and will be happy to remove it from here. :)

Reply:
A best practice of Maven is CoC, so I would ask you to not repeat implicit defaults without a good reason. Thanks.